Client/sdk/game/CStreaming.h

```diff
@@ -11,12 +11,37 @@
 
 #pragma once
 
+struct CStreamingInfo
+{
+    WORD  prevId;
+    WORD  nextId;
+    WORD  nextInImg;
+    BYTE flg;
+    BYTE archiveId;
+    DWORD offsetInBlocks;
+    DWORD sizeInBlocks;
+    DWORD loadState;
```

Comment:
I think we discourage using `WORD` `BYTE` `DWORD` etc now, and prefer proper types etc

Reply:
Fixed

Reply:
Perhaps uint16_t, etc.. should be used? Since, remember, `unsigned int` can be 2 bytes as well on a compiler, since its not standardized.

Reply:
It's dependents from processor architecture. For 16 bits `int` will have size 2 bytes, for 32 bits - 4 bytes. MTA client compiled for 32 bit architecture always.

Reply:
Well, `int` was a bad example, but the only one applicable here.
A better example would be `long` which, depending on the memory model used in the compiler, is either 32 or 64 bits. On MSVC, which uses LLP64, its 32, but iirc on GCC it's 64. Not

Reply:
Hmm... Changed to uint8, uint16, uint32